bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,337 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dirigera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="text">
+		<item-type>String</item-type>
+		<label>Simple Text</label>
+	</channel-type>
+	<channel-type id="text-veto">
+		<item-type>String</item-type>
+		<label>Veto Text</label>
```

Comment:
For a user this would be a weird label. It weould probably be overriedn everywhere, but better to rename to something else,

Reply:
Sure, but I think I took care all these initial labels are overwritten.